mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -288,29 +310,26 @@ def _load_mesh(self):  # {{{
             raise IOError('No MPAS-O restart file found: need at least one '
                           'restart file for MOC calculation')
         ncFile = netCDF4.Dataset(restartFile, mode='r')
-        dvEdge = ncFile.variables['dvEdge'][:]
-        areaCell = ncFile.variables['areaCell'][:]
+        self.dvEdge = ncFile.variables['dvEdge'][:]
+        self.areaCell = ncFile.variables['areaCell'][:]
         refBottomDepth = ncFile.variables['refBottomDepth'][:]
-        latCell = np.rad2deg(ncFile.variables['latCell'][:])
+        self.latCell = np.rad2deg(ncFile.variables['latCell'][:])
+        self.cellsOnEdge = ncFile.variables['cellsOnEdge'][:]-1
         ncFile.close()
+
         nVertLevels = len(refBottomDepth)
-        refTopDepth = np.zeros(nVertLevels+1)
-        refTopDepth[1:nVertLevels+1] = refBottomDepth[0:nVertLevels]
-        refLayerThickness = np.zeros(nVertLevels)
-        refLayerThickness[0] = refBottomDepth[0]
-        refLayerThickness[1:nVertLevels] = \
-            (refBottomDepth[1:nVertLevels] -
-             refBottomDepth[0:nVertLevels-1])
-
-        return dvEdge, areaCell, refBottomDepth, latCell, nVertLevels, \
-            refTopDepth, refLayerThickness
+        self.refTopDepth = np.zeros(nVertLevels+1)
+        self.refTopDepth[1:nVertLevels+1] = refBottomDepth[0:nVertLevels]
+        self.nVertLevels = nVertLevels
+
         # }}}
 
     def _compute_velocity_climatologies(self):  # {{{
         '''compute yearly velocity climatologies and cache them'''
 
         variableList = ['timeMonthly_avg_normalVelocity',
-                        'timeMonthly_avg_vertVelocityTop']
+                        'timeMonthly_avg_vertVelocityTop',
+                        'timeMonthly_avg_layerThickness']
```

Comment:
I add `layerThickness` so we can compute `layerThicknessEdge` from it later on.  This way, we're not relying on `refLayerThickness`.